code/modules/paperwork/paper.dm

```diff
@@ -437,3 +437,83 @@
 
 /obj/item/paper/crumpled/bloody
 	icon_state = "scrap_bloodied"
+
+//Evil faxes
+/obj/item/paper/evilfax
```

Comment:
is this just an admin only thing?

Reply:
Yep. If someone faxes Centcom WGW or something similarly stupid, admins can smite them with these.